### PR TITLE
Demisto Rest API: JSON parse body only if exist

### PR DESCRIPTION
**Pull request by ilappe**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45136)

## Description
Try to parse the body as json only if is exist

### CODE REVIEW

File: Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js [hunk 1]
```diff
@@ -120,7 +120,11 @@ switch (command) {
         sendRequest('GET','user');
         return 'ok';
     case 'demisto-api-post':
-        var body = JSON.parse(args.body);
+        if(args.body)
+            var body = JSON.parse(args.body);
+        else
+            logDebug('The body is empty.')
+
         return sendRequest('POST',args.uri, args.body);
```
Comment (ilappe) [inline Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js:128]: @yaakovi 
we parse the body but didn't send the body variable to the request 
is this by mistake?

Comment (yaakovi) [reply Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js:128]: don't think so, please test this behavior

**Review by yaakovi — COMMENTED** [review]
please test this change (enhance the test playbook)

File: Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js (hunk 1 above)
Comment (ilappe) [reply Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js:128]: checked 
it fail if we send the body var

**Review by yaakovi — APPROVED** [review]

**Pull request merged**